boards/clue_nrf52840/Makefile

```diff
@@ -17,25 +17,12 @@ KERNEL_WITH_APP=$(TOCK_ROOT_DIRECTORY)/target/$(TARGET)/debug/$(PLATFORM)-app.el
 .PHONY: program program-apps
 
 program: $(TOCK_ROOT_DIRECTORY)target/$(TARGET)/debug/$(PLATFORM).bin
-	ifndef PORT
```

Comment:
Can you separate out this Makefile changes to a PR to master?

Reply:
Sorry for that, I did not notice the Makefile was commited.